### PR TITLE
Feature/travelers repository and test

### PR DESCRIPTION
**Pull request by Cyanne-Jones**
This branch added:

- TravelersRepository class file
- Accompanying test file to assert all functionality is performing as expected
- Sample data files pulled from API data to use for testing

All tests are passing and this branch was checked for misspellings and syntax inconsistencies

### CODE REVIEW

File: data/travelers.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+//5 travelers
+
+const travelersSampleData = [
```
Comment (CorCanavan) [inline data/travelers.js:3]: love how you have the sample data in a separate file!

File: test/TravelersRepository-test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  beforeEach(() => {
+    travelersRepository = new TravelersRepository(travelersSampleData);
+  })
+
+  it('Should be a function', function() {
+    expect(TravelersRepository).to.be.a('function');
+  });
+
+  it('should be an instance of UserRepository', () => {
```
Comment (CorCanavan) [inline test/TravelersRepository-test.js:18]: I think you meant TravelersRepository here on line 18 instead of UserRepository :)

Comment (Cyanne-Jones) [reply test/TravelersRepository-test.js:18]: Such a good catch!! I will change this ASAP!

**Review by CorCanavan — APPROVED** [review]
Looks great! Only note is that minor typo to change UserRepository to TravelersRepository on line 18 of the TravelersRepository-test file. Awesome work!

**Pull request merged**